lib/lib_rss.php

```diff
@@ -515,13 +515,9 @@ function validateEmailAddress(string $email): bool {
  * @param string $content is the text we want to parse
  */
 function lazyimg(string $content): string {
-	return preg_replace([
-			'/<((?:img|iframe)[^>]+?)src="([^"]+)"([^>]*)>/i',
-			"/<((?:img|iframe)[^>]+?)src='([^']+)'([^>]*)>/i",
-		], [
-			'<$1src="' . Minz_Url::display('/themes/icons/grey.gif') . '" data-original="$2"$3>',
-			"<$1src='" . Minz_Url::display('/themes/icons/grey.gif') . "' data-original='$2'$3>",
-		],
+	return preg_replace(
+		'/<((?:img|iframe)[^>]+?)([^>]*)>/i',
+		'<$1 loading="lazy"$2>',
```

Comment:
We need to check upstream that we filter out existing `loading` attributes (that might already be the case), to make sure we do not end up with two identical attributes, which would be invalid